scale-encode/src/lib.rs

```diff
@@ -147,6 +147,7 @@ mod impls;
 
 pub mod error;
 
+pub use alloc::vec::Vec;
```

Comment:
Aah, exposing this is a clever way to get the type you need in the macro :)

One tiny thing; let's add `#[doc(hidden)]` and a comment like "This is exported for generated derive code to use, to be compatible with std or no-std as needed"

Reply:
Added 👍